pretty_gpx/main.py

```diff
@@ -5,7 +5,10 @@
 import tempfile
```

Comment:
Hi @JulesL2,

Thanks for your input! I’ve actually been working on incorporating the city mode into the GUI as well. I’m using the page element of NiceGUI to create a sidebar menu that allows users to switch between the city and mountain modes, with each rendering mode having its own dedicated page.

I’m planning to open a PR tomorrow or over the weekend

Apologies for any redundancy in effort—I appreciate your contribution!

Reply:
Hi @ThomasParistech

Nice update 😀 looking forward to have the PR merged 😄

Reply:
By the way, when running the app on the trail des 4 chateaux, i got this error. Does it also raise the error when you run it (after emptying the cache)?

TypeError: Unexpected member type <class 'overpy.RelationNode'> not in [RelationWay, RelationRelation]